device-types/Juniper/EX4100-48P.yaml

```diff
@@ -0,0 +1,135 @@
+---
+manufacturer: Juniper
+model: EX4100-48P
+slug: ex4100-48p
+u_height: 1
```

Comment:
Please add the following missing values, if you have them:

1. `part_number`
2. `is_full_depth`
3. `airflow`
4. `weight` and `weight_unit`

If you do not have, or cannot find any of these values just let me know. Often you canfind all of these on the manufacturers datasheet for this product